jsapp/js/account/organization/membersInviteQuery.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  useQuery,
+  useQueryClient,
+  useMutation,
+} from '@tanstack/react-query';
+import {fetchPost, fetchGet, fetchPatchUrl, fetchDeleteUrl} from 'js/api';
+import {type OrganizationUserRole, useOrganizationQuery} from './organizationQuery';
+import {QueryKeys} from 'js/query/queryKeys';
+import {endpoints} from 'jsapp/js/api.endpoints';
+import type {FailResponse} from 'jsapp/js/dataInterface';
+import {type OrganizationMember} from './membersQuery';
+import {type Json} from 'jsapp/js/components/common/common.interfaces';
+
+/*
+ * NOTE: `invites` - `membersQuery` holds a list of members, each containing
+ * an optional `invite` property (i.e. invited users that are not members yet
+ * will also appear on that list). That's why we have mutation hooks here for
+ * managing the invites. And each mutation will invalidate `membersQuery` to
+ * make it refetch.
+ */
+
+/*
+ * NOTE: `orgId` - we're asserting it is not `undefined` in code below,
+ * because the parent query (`useOrganizationMembersQuery`) wouldn't be enabled
+ * without it. Plus all the organization-related UI (that would use this hook)
+ * is accessible only to logged in users.
+ */
+
+/**
+ * This is the same list as `OrganizationInviteStatusChoices` from
+ * `kobo/apps/organizations/models.py`.
+ */
```

Comment:
```suggestion
/**
 * The source of truth of statuses are at `OrganizationInviteStatusChoices` in
 * `kobo/apps/organizations/models.py`. This enum should be kept in sync.
 */
```

Let's be clear on which of two is the source of truth